src/main/scala/viper/server/ViperConfig.scala

```diff
@@ -83,4 +101,29 @@ class ViperConfig(args: Seq[String]) extends ScallopConf(args) {
     noshort = false,
     hidden = false
   )
-}
\ No newline at end of file
+
+  val nThreads: ScallopOption[Int] = opt[Int](
+    name = "nThreads",
+    descr = s"Maximal number of threads that should be used (not taking threads used by backend into account)\n" +
+      s"Values below ${DefaultVerificationExecutionContext.minNumberOfThreads} (the minimum) will be set to the minimum.\n" +
+      s"The default value is the maximum of ${DefaultVerificationExecutionContext.minNumberOfThreads} and the number of available processors",
+    default = Some(Math.max(DefaultVerificationExecutionContext.minNumberOfThreads, Runtime.getRuntime.availableProcessors())),
+    noshort = true
+  )(singleArgConverter(input => {
+    // parse option as int and check bounds
+    val n = input.toInt
+    n match {
+      case n if n < DefaultVerificationExecutionContext.minNumberOfThreads => DefaultVerificationExecutionContext.minNumberOfThreads
+      case n => n
+    }
+  }, numberHandler("Int")))
+
+  /**
+    * Exception handling
+    */
+  /**
+    * Epilogue
```

Comment:
What are these two comments referring to? :-)

Reply:
This is copy pasted from some other Scallop subclass used in Gobra or Silicon. It refers to the steps that are performed to specify resp. check the inputs